### DISCUSSION TITLE
Distribute founds to contributors

### DISCUSSION THREAD

**Discussion by wipfli** [maplibre/maplibre#10] [General governance]
As of Feb 11, 2022, we have 3.6 kUSD on our OpenCollective account, see https://opencollective.com/maplibre

We should distribute this money to MapLibre contributors who are not already payed by companies for their contributions.

I will list some contributors that come to mind for receiving founds. Feel free to add more people. Feel free to show your support and say thanks by adding some emojis.

**Comment by wipfli** [comment]
@birkskyum

**Comment by wipfli** [comment]
@HarelM

**Comment by wipfli** [comment]
@ntadej

**Comment by wipfli** [comment]
@astridx

**Comment by wipfli** [comment]
@xabbu42

**Reply by xabbu42** [reply]
most of my contributions are part of my paid work already, so no need here, but thanks anyway :-).

**Comment by wipfli** [comment]
@vanilla-lake

**Reply by vanilla-lake** [reply]
My contributions were done as part of my job, so no need for me. But thanks a lot for asking! 😃

**Comment by HarelM** [comment]
@wipfli

**Comment by birkskyum** [comment]
@jutaz

**Comment by birkskyum** [comment]
@prozessor13

**Reply by wipfli** [reply]
First, thanks Max for all the energy you put into the 3D pull request. You can comment yourself, but probably Toursprung covered you already (thanks Toursprung:)

**Reply by prozessor13** [reply]
Thx for asking!, but yes, i did all work as part of my normal work at Tourspung. So thx Toursprung!

**Comment by nyurik** [comment]
Proposed distributions, will distribute in the next few days unless there is an objection. Total - $4300, which will bring the balance to $4.67.

| name | amount |
|---|---|
@birkskyum | 2000
@HarelM	| 1000
@wipfli 	| 800
@ntadej	| 400
@astridx	| 100
